packages/dd-trace/src/llmobs/telemetry.js

```diff
@@ -102,10 +102,18 @@ function recordLLMObsSpanSize (event, eventSize, shouldTruncate) {
   llmobsMetrics.distribution('span.size', tags).track(eventSize)
 }
 
+function recordDroppedPayload (numEvents, eventType, error) {
+  if (eventType !== 'span' && eventType !== 'evaluation_metric') return
```

Comment:
```suggestion
  if (eventType !== 'span' || eventType !== 'evaluation_metric') return
```
missed this before but i think this is right 😛

Reply:
wait we're trying to return early if the event type is neither `span` or `evaluation_metric`, is && not the correct operator here?